libraries/radspreadstreamprocessing/features/culture-spesific-predefined-formats.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Culture Specific Predefined Formats 
+page_title: Culture Specific Predefined Formats
+description:  
+slug: radspreadstreamprocessing-features-culture-spesific-predefined-formats
+tags: culture, format, currency, number
+platforms: core,blazor,xamarin,winui, wpf, ajax, mvc, silverlight
```

Comment:
This is not needed. SpreadStreamProcessing is available for all platforms except Silverlight and the bar is set up on the library folder's level.

Reply:
OK will remove it